prow/spyglass/spyglass.go

```diff
@@ -304,6 +313,50 @@ func (s *Spyglass) RunToPR(src string) (string, string, int, error) {
 	}
 }
 
+// ExtraLinks fetches started.json and extracts links from metadata.links.
+func (sg *Spyglass) ExtraLinks(src string) ([]ExtraLink, error) {
+	artifacts, err := sg.FetchArtifacts(src, "", 1000000, []string{"started.json"})
```

Comment:
We don't fetch artifacts anywhere else?

Reply:
Not in the context of the root spyglass page — in order to be speedy, we try to avoid doing things that take a long time. As such, while we do make several requests to GCS, we don't actually _retrieve_ anything (and, in as much as is possible, we should continue to not do so).

To also avoid doing so here we would need to fetch and inject the links after page load, which is a decent amount of complexity and perhaps slightly awkward UX for probably not much win.